internal/policies/apparmor/apparmor.go

```diff
@@ -245,22 +228,109 @@ func unloadAllRules(ctx context.Context, apparmorParserCmd []string, loadedPolic
 	if err != nil {
 		return err
 	}
-	policies, err := policiesFromFiles(ctx, apparmorParserCmd, loadedPoliciesFile, filesToUnload, true)
+	policies, err := m.policiesFromFiles(ctx, filesToUnload, true)
 	if err != nil {
 		return err
 	}
 
-	if err := unloadPolicies(ctx, apparmorParserCmd, policies); err != nil {
+	if err := m.unloadPolicies(ctx, policies); err != nil {
 		return err
 	}
 
 	// Unloading succeeded, remove apparmor policy dir
 	if err := os.RemoveAll(apparmorPath); err != nil {
-		log.Warningf(ctx, i18n.G("Couldn't remove apparmor directory: %v"), err)
```

Comment:
so, as the test are still passing, it means that no test was entering this failure, is it doable at all or previous checks prevents us to be able to easily test this?